docs/cmd/kn_service_update.md

```diff
@@ -41,6 +41,9 @@ kn service update NAME [flags]
   -p, --port int32               The port where application listens on.
       --requests-cpu string      The requested CPU (e.g., 250m).
       --requests-memory string   The requested memory (e.g., 64Mi).
+      --tag strings              Tag revisions, format: --tag revision=tag , example: --tag echo-abcde=current (can be specified multiple times). Use identifier @latest to refer to latest ready revision, for e.g.: --tag @latest=new (@latest can be used only once with --tag flag).
+      --traffic strings          Set traffic percentage, format: --traffic revision=percent , example: --traffic echo-abcde=50) (can be specified multiple times). Use identifier @latest to refer to latest ready revision, for e.g.: --traffic @latest=100 (@latest can be used only once with --traffic flag).
+      --untag strings            Untag revision, format: --untag tag , example: --untag current
```

Comment:
please no examples here, otherwise the description gets too long. It's meant as a reference.

Reply:
Ack!

Reply:
Or as shown below which would be more complete:

> Set traffic distribution in the format "revOrTag1=percent,revOrTag2=percent,..." where name can be a revision, a tag or '@latest' for the latest ready revision. This option can be given multiple times, summing up to 100%.